manager/plugin/plugin.go

```diff
@@ -172,8 +172,9 @@ func (m *vatzPluginManager) Uninstall(name string) error {
 			return err
 		}
 		if running {
-			log.Error().Str("module", "plugin").Err(err).Msgf("Plugin %s is currently running, Please, kill plugin first.", name)
-			return nil
+			log.Error().Str("module", "plugin").Err(err).Msgf("Plugin %s is currently running, Please, stop plugin first.", name)
+			errorMessage := fmt.Sprintf("Please, stop plugin: %s first.", name)
```

Comment:
Suggest to use `fmt.Errorf`, too.

Reply:
@rootwarp 
Update as you mentioned, let me know if there's more to update or fix. 
appreciate it!!